TCMSApp/src/client/app/tests/tests.controller.js

```diff
@@ -10,7 +10,13 @@
     function TestsController(logger, FakeTestsFactory, $uibModal) {
         var vm = this;
 
+        vm.stepList = [1,2];
+        vm.addStep = addStep;
+        vm.delStep = delStep;
+
         vm.trigger = 0; // suite -> testCases
```

Comment:
as I see this is the index of the current (selected) test suite?
We may try to write code without triggers.

I will try to explain.
In ideal world we might have some model in the service. and something like currentTestSuite.
I think I will do this as a separate task right now.